src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-resources.mdx

```diff
@@ -8,21 +8,117 @@ metaDescription: Here are some tips for working with OpenTelemetry resources and
+Data received is associated with an entity through a process called [entity synthesis](/docs/entities/synthesis.md)
+whereby the resource attributes present are matched to rules which determine the type of entity the
+data will be associated with. For data sourced from OpenTelemetry, the entity synthesis rules are
+designed to respect the OpenTelemetry [resource semantic conventions](https://opentelemetry.io/docs/specs/semconv/resource).
 
-* `service.* attributes`
-  * `service.name` attribute is required to associate your resource with an entity in the UI 
-  * `service.instance.id` is required for certain panes to light up
-* `telemetry.sdk.language=java` is required to see data in the JVM section 
+The resource semantic conventions define different bodies of conventions for
+describing different types of entities. For example, the
+[service](https://opentelemetry.io/docs/specs/semconv/resource/#service) and
+[host](https://opentelemetry.io/docs/specs/semconv/resource/host/)
+conventions define the resource attributes which describe a service or host instance,
+respectively.
 
-## Organize your entities in the UI using tags [#tags]
+## Supported entity types
 
-You can use tags to organize and filter your entities in the UI. [Tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/) are key-value pairs, for example `team: operations`, added to various sets of data, like monitored apps and hosts. Certain important attributes are automatically available as tags, such as account ID; you can also add your own custom tags [directly in the UI](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-tags), or with one of the methods below.
+Described here are the New Relic entity types which are supported when using
+OpenTelemetry instrumentation. Included for each supported entity type are:
 
-Use the [resource processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourceprocessor) to add resource attributes prefixed with `tags`, which will result in a tag on your entity in New Relic. For example, adding the attribute `tags.mytag=myvalue` will result in the entity tag `mytag=myvalue`. Here’s an example configuration:
+1. The resource attributes required for entity synthesis.
+2. Recommended attributes that drive certain aspects of the New Relic UI.
+3. Attributes which become [entity tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/),
+   when present.
 
-<img width="254" alt="Screen Shot 2022-08-15 at 2 22 34 PM" src="https://user-images.githubusercontent.com/48657837/184720791-d2040326-55e6-4932-a4da-87d5ed4e6801.png"/>
+### Services
```

Comment:
> If you want this to show up in the in-page navigation

I conceded and did not promote everything to h2. I think the right hand nav is not too terrible as I've organized it.

*  Supported entity types
* Supported entity relationships
* Adding custom tags to an entity

Reply:
> The right hand nav is a pretty standard pattern in technical documentation. But I think NR's implementation has a lot to be desired.

I think all it would take would be two levels of headers to be included instead of one.

Reply:
@jack-berg @alanwest I agree that the right nav can and should be improved. I believe our eng team is currently working on the ability to display H3s in the right nav. In the meantime, let's keep writing strong titles for headers keeping the right-nav presentation in mind.